internals/maintenance_scripts.py

```diff
@@ -1132,3 +1132,17 @@ def get_template_data(self, **kwargs):
       count += len(changed_stages)
 
     return f'{count} rollout_milestone fields migrated'
+
+
+class ResetOutstandingNotifications(FlaskHandler):
+  """Reset the FeatureEntry.outstanding_notifications counter for all features"""
+
+  def get_template_data(self, **kwargs) -> str:
+    self.require_cron_header()
+    notified_features = FeatureEntry.query(
+      FeatureEntry.outstanding_notifications >= 1
+    ).fetch()
+    for f in notified_features:
```

Comment:
You may want to add a log statement here like `Resetting outstanding_notifications for feature %d from %d to 0.` as well for auditing reasons